packages/docs/docs/getting-started/what-can-you-build-with-unirep.md

```diff
@@ -0,0 +1,29 @@
+---
+
+title: what can you build with unirep
+---
+
+# 🏗️ What Can You Build With UniRep
+
+UniRep is a generalized protocol that allow developers to build many different kinds of applications while keep user data private. Here are a few examples to illustrate the broad range of applications that could be built:
```

Comment:
actually data is public but the user is anonymous.
so it is more like keeping user's identity private (or keeping users anonymous)